### DISCUSSION TITLE
Drop Non-Numeric

### DISCUSSION THREAD

**Discussion by memberA** [grafana/grafana#67801] [Alerting]
In this mode all non-numeric values (null, NaN or +-Inf) in the input series are filtered out before executing the reduction function.

why it still shows me 0 and NaN?

![image](https://user-images.githubusercontent.com/91407250/236137449-d29e5c85-6bd4-49b6-b03f-29e2c5f6d6b9.png)
![image](https://user-images.githubusercontent.com/91407250/236137495-172bd4aa-eecb-4203-bfad-444d3e3341fd.png)

**Comment by memberA** [comment]
does anyone know how to make it not show in Alerts `0` and `NaN` ?

**Comment by memberA** [comment]
Ok. I solve the problem in this way
![image](https://github.com/grafana/grafana/assets/91407250/e0eccc01-108f-4ee3-af17-3b75dfa41a94)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). It looks like this discussion has an answer, so I will close it. Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you would like to continue the discussion around it.
